webv2/helpers/helpers.go

```diff
@@ -79,6 +79,24 @@ func createDatabase(ctx context.Context, uri string) error {
 				SchemaConversionObject JSON NOT NULL,
 				CreateTimestamp TIMESTAMP NOT NULL,
 			  ) PRIMARY KEY(VersionId)`,
+			  `CREATE TABLE JobExecutionData (
+				MigrationJobId STRING(50) NOT NULL,
+				SpannerDatabaseName STRING(50) NOT NULL,
+				AggMonitoringResources JSON,
+				IsShardedMigration BOOL NOT NULL,
+				CreateTimestamp TIMESTAMP NOT NULL,
```

Comment:
We should maintain a state of the job here

Reply:
We should also soft delete jobs, not really clear them out. The state can double up for this if required

Reply:
Probably also need an updated time here

Reply:
This is not a `JobDetails` table. It only contains the execution data for job i.e artefacts that are relevant for job execution. 

Job status and associated semantics will be relevant to a `JobDetails` and will add more details such as `status`, `type`, `config` etc and is out of scope of the current PR.

Reply:
Added `UpdatedAt` field to both the tables